app/src/main/java/com/example/triptracker/view/home/HomeUtils.kt

```diff
@@ -148,6 +155,14 @@ fun DisplayItinerary(
                     // overview with the selected itinerary highlighted and the first pinned places
                     onClick()
```

Comment:
Doesn't this one have to be removed if you have the combined clickable onClick? (see line 159)

Reply:
Yes in fact thanks !